### PR TITLE
Style breadcrumb style

### PR DESCRIPTION
**Pull request by ArthurTriis1**
## What's the purpose of this pull request?
This PR styles the Breadcrumb component

## How it works? 
Add a stylesheet for Breadcrumb and implement a style like the [Figma Prototype](https://www.figma.com/file/wS0Kek1aSzIRl20jdtnSsL/Alpha-Version?node-id=0%3A1)

Screen Shot 2021-12-30 at 16 18 41

### CODE REVIEW

**Review by israelswf — APPROVED** [review]
LGTM ✨

**Review by hellofanny — APPROVED** [review]
Well done guys!!! 🚀

File: src/components/ui/Breadcrumb/Breadcrumb.tsx [hunk 1]
```diff
@@ -13,17 +15,17 @@ export interface BreadcrumbProps extends UIBreadcrumbProps {
...
 
     return `/${parsedUrl[parsedUrl.length - 1]}`
   }
 
   return (
-    <UIBreadcrumb divider={divider}>
+    <UIBreadcrumb divider="">
       <Link aria-label="home" href="/">
-        <HouseIcon />
+        <HouseIcon size={18} />
```
Comment (renatamottam) [inline src/components/ui/Breadcrumb/Breadcrumb.tsx:28]:
```suggestion
        <HouseIcon size={18} weight="bold" />
```

**Review by renatamottam — APPROVED** [review]
Great work, guys!

File: src/components/ui/Breadcrumb/Breadcrumb.tsx [hunk 2]
```diff
@@ -13,17 +15,17 @@ export interface BreadcrumbProps extends UIBreadcrumbProps {
...
+function Breadcrumb({ breadcrumbList }: BreadcrumbProps) {
   const buildUrl = (url: string) => {
     const parsedUrl = url.split('/')
 
     return `/${parsedUrl[parsedUrl.length - 1]}`
   }
 
   return (
-    <UIBreadcrumb divider={divider}>
+    <UIBreadcrumb divider="">
```
Comment (eduardoformiga) [inline src/components/ui/Breadcrumb/Breadcrumb.tsx:26]: I noticed that when we pass a divider here something strange happens

Screen Shot 2021-12-30 at 20 28 57

**Review by eduardoformiga — CHANGES_REQUESTED** [review]
Looks good! 
I notice some behaviors that we need to double-check if it is desired.

- Line breaks when the text is long on mobile.
- Visited links with different colors (`Desks` is blue and `Office` is purple below. I'm not sure if it is desired).

Screen Shot 2021-12-30 at 20 25 54

When we pass a `divider` here something strange happens:
Screen Shot 2021-12-30 at 20 28 57

File: src/components/ui/Breadcrumb/Breadcrumb.tsx (hunk 2 above)
Comment (ArthurTriis1) [reply src/components/ui/Breadcrumb/Breadcrumb.tsx:26]: The default divider of this component is "/", this need be changed in faststore, but to our use divider="" resolved this problem

**Review by lucasfp13 — APPROVED** [review]
LGTM!

**Pull request merged**